astropy/io/fits/hdu/hdulist.py

```diff
@@ -166,14 +186,34 @@ def fitsopen(name, mode='readonly', memmap=None, save_backup=False,
     else:
         lazy_load_hdus = bool(lazy_load_hdus)
 
+    if use_fsspec is None:
```

Comment:
I think if the parameter is not given, it should default to the config one -- and that can be `None` to indicate the behaviour here.